api/versions.go

```diff
@@ -62,10 +63,11 @@ func getApiVersions() []ApiVersion {
 	v1_1 := newVersion1_1(v1_0)
 	v1_2 := newVersion1_2(v1_1)
 	v1_3 := newVersion1_3(v1_2)
+	v1_4 := newVersion1_4(v1_3)
 	v2_0 := newVersion2_0()
 	v2_1 := newVersion2_1(v2_0)
 
-	return []ApiVersion{v1_0, v1_1, v1_2, v1_3, v2_0, v2_1}
+	return []ApiVersion{v1_0, v1_1, v1_2, v1_3, v1_4, v2_0, v2_1}
```

Comment:
You don't need to add a new version for this, since it's not changing exsiting behavior. Also, can we expose this as part of th 2.x API instead of 1.x?